py3status/modules/sysdata.py

```diff
@@ -260,6 +264,11 @@ def post_config_hook(self):
         if self.init["stat"]:
             self.cpus = {"cpus": self.cpus, "last": {}, "list": []}
 
+        if self.init["zone_sensor"]:
+            self.zone_sensor = re.compile(self.init["zone_sensor"][0])
+        else:
+            self.zone_sensor = re.compile(self.zone_sensor)
```

Comment:
```bash
$ python sysdata.py 
Exception in `test_module` post_config_hook() (KeyError) module.py line 136.
Traceback
KeyError: 'zone_sensor'
  File "/home/chris/src/py3status/py3status/module.py", line 136, in prepare_module
    self.module_class.post_config_hook()
  File "sysdata.py", line 267, in post_config_hook
    if self.init["zone_sensor"]:

terminating module test_module
```
I think you are trying to do a simple `if self.init["cpu_temp"] --> compile self.zone_sensor`.

Reply:
Yeah, you are completely right, thanks. Fixed and rebased.